docs/procedures/general_checks.md

```diff
@@ -0,0 +1,110 @@
+# General Checks
+
+## Before Measuring
+
+Here are a few checks to perform before starting measurements.
+These are updated as much as possible for Run 3 values.
+
+- [ ] <details class="nodeco"><summary>Make Sure Intensity Is below $10^{10}$ppb until Local Corrections Are In</summary>
+      <p> After local corrections, we can move to having 3 bunches at $10^{10}$ppb evenly spaced along the ring.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Mask the Appropriate BLMs</summary>
+      <p> It is possible tp "mask" some of the BLMs, which means making sure they won't trigger any beam dump. They are essentially ignored in the interlocked system when masked.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Turn Off the Landau Octupoles</summary>
+      <p> Unless you specifically need them on for your measurements (ampdet & co).
+      These are the MO elements.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Correct the Machine Chromaticity to a Few Units</summary>
+      <p> In the past we've used $2$, which we will most likely keep using in Run 3.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Mask the ATLAS BCM</summary>
```

Comment:
We cannot change this setting from the CCC. The only way is through contacting the ATLAS control room. Talk to the EIC. At least in the beginning we should double check that the multiturn is actually giving the right indication.

Reply:
Done in [a5c6908](https://github.com/pylhc/pylhc.github.io/pull/69/commits/a5c6908372ef4055d5b99ee516d3330401ec7eae)